components/shared/right-sidebar/SidebarSocialMedia.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Link from 'next/link';
+import SocialMediaLinks from './SocialMediaLinks'
+import PlusIcon from '../Icons/PlusIcon';
+
+const SidebarSocialMedia = () => {
+  return (
+    <div className='flex flex-col gap-4'>
+      <Link href='/profile/edit-profile/update-socials'>
+        <button className="paragraph-3-medium bg-black-600 flex w-full flex-row items-center justify-center rounded py-3 font-bold text-white">
+          <PlusIcon className='fill-primary-500'/>
+          <span className='ml-2'>Add a new link</span>
+        </button>
```

Comment:
Same adjustments here- and as you **probably** notice we're getting some repeated code that would likely do well as a reusable component later.